src/color.c

```diff
@@ -25,6 +25,11 @@
 #include <time.h>
 #include <unistd.h>
 
+#if __FreeBSD__
```

Comment:
I think we shouldn't limit this to just FreeBSD.  For starters, macOS uses it too.  But also, in the same way that `bfs` brings the GNU-style colors to all platforms, I think it should bring the BSD-style ones to all platforms, if that's what's in your environment.